frontend/src/component/common/Dialogue/Dialogue.tsx

```diff
@@ -5,12 +5,42 @@ import {
     DialogActions,
     DialogContent,
     DialogTitle,
+    styled,
 } from '@mui/material';
 
 import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
-import { useStyles } from './Dialogue.styles';
 import { DIALOGUE_CONFIRM_ID } from 'utils/testIds';
 
+const StyledDialog = styled(Dialog)(({ theme, maxWidth }) => ({
+    '& .MuiDialog-paper': {
+        borderRadius: theme.shape.borderRadiusLarge,
+        maxWidth: !maxWidth ? theme.spacing(85) : undefined,
+        backgroundColor: 'transparent',
```

Comment:
This fixes the issue with the extra white pixels at the borders when using border radius.